source/contracts/factories/WindowParticipationTokenFactory.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity 0.4.17;
+
+
+import 'libraries/Delegator.sol';
+import 'reporting/IWindowParticipationToken.sol';
+import 'reporting/IReportingWindow.sol';
+import 'IController.sol';
+
+
+contract WindowParticipationTokenFactory {
```

Comment:
Consider: Rename `Window*` to `ReportingWindow*`.  I know it makes these things get a bit long, but `Reporting` is a fairly important piece of information in the name IMO.

Reply:
I was going to do this but it puts us over the `string` limit, so for the sake of that I'll keep it the same

Reply:
😡

Reply:
What about `ReportingParticipationToken` in that case?  I feel like if we have to cut `Reporting` or `Window` I would rather cut `Window`.  However, I feel like there was a reason we didn't go with that name in the first place?

Reply:
There was some discussion but nothing too important. `ReportingParticipationToken` sounds good!

Reply:
Heh, this also puts it over the character limit.